psalm.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<psalm
+        errorLevel="2"
```

Comment:
```suggestion
        errorLevel="4"
```

Reply:
I've fixed some errors and brought the errorLevel down to 2, we should keep it this way. In the future we should aim to decrease the psalm error reporting levels on all packages.